### DISCUSSION TITLE
Set up public asynchronous chat for armi team

### DISCUSSION THREAD

**Discussion by ntouran** [terrapower/armi#523] [Ideas]
We've had some requests to set up a way to chat quickly with the team for fast iteration.

**Comment by ntouran** [comment]
Options: https://rigorousthemes.com/blog/best-open-source-discord-alternatives/

* Discord seems pretty common

**Reply by scottyak** [reply]
It would be ideal if the chat supports signing in with a GitHub account (e.g. Gitter). It's just one less thing to sign up for. Personally, I use Discord for gaming/crypto-related stuff, and I'd prefer not to associate my Discord username with my professional identity.

**Reply by jakehader** [reply]
I like the idea of something that supports different threads/channels, supports uploading files, and possibly voice channels for support/community engagement. I can see how having something that links to the GitHub account (like @scottyak mentioned) would be good to minimize the number of extra accounts people need.

I just logged into Gitter and it seems (at least on mobile) that it's just one big chat and that might not be ideal for different types of conversations.

Matrix-based applications seem to be pretty good as they integrate directly with GitHub account logins. Element is the first one on the list, but I like the visual appeal of Cinny - https://cinny.in/#features

Maybe we can start by looking through the Matrix apps that people have made and making suggestions. I don't think that Cinny supports video/voice currently but I'm not sure how much of a need that really is or how much we'd use that.

**Reply by scottyak** [reply]
Cinny does look nice! Agree that having one big chat is not ideal.
